src/hooks/members/useMemberFilters.ts

```diff
@@ -58,7 +59,9 @@ const useMemberFilters = ({
 
   const filteredByPermissions = useMemo(() => {
     const databaseDomainIds = new Set(
-      nativeDomainIds.map((id) => `${colonyAddress}_${id}`),
+      selectedTeam
+        ? [selectedTeam.id]
```

Comment:
I feel like this should probably be of the same format as the other option: `${colonyAddress}_${id}`. What do you think?

Reply:
@chmanie It contains the same format as you described. It's taken from the colony context. Let me know if it can stay like this, or if you want me to change it anyway.

Reply:
Aaah right I forgot that these are stored in the db that way. Don't mind me then, sorry to interrupt :)